### PR TITLE
Feature:  adding migrations to the project

### PR DESCRIPTION
**Pull request by eapg**
* Adding migrations to the project
* Adding first migration version - db script for tables and contrains

### CODE REVIEW

File: migrations/README [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is a database migration repository.
```
Comment (reynoldmorel) [inline migrations/README:1]: Can you move these docs to the parent README?

Can you try to add the commands that you normally use to make it work?

File: migrations/migrate.cfg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[db_settings]
+# Used to identify which repository this database is versioned under.
+# You can use the name of your project.
+repository_id=ea restaurant migrations
```
Comment (reynoldmorel) [inline migrations/migrate.cfg:4]: Could you avoid using white spaces? Use underscore instead

File: migrations/migrate.cfg [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# The name of the database table used to track the schema version.
+# This name shouldn't already be used by your project.
+# If this is changed once a database is under version control, you'll need to
+# change the table name in each database too.
+version_table=migrate_version
```
Comment (reynoldmorel) [inline migrations/migrate.cfg:10]: You meant `migrate_versions`?

File: migrations/migrate.cfg [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+# Databases in this list MUST compile successfully during a commit, or the
+# entire commit will fail. List the databases your application will actually
+# be using to ensure your updates to that database work properly.
+# This must be a list; example: ['postgres','sqlite']
+required_dbs=[]
+
+# When creating new change scripts, Migrate will stamp the new script with
+# a version number. By default this is latest_version + 1. You can set this
+# to 'true' to tell Migrate to use the UTC timestamp instead.
+use_timestamp_numbering=False
```
Comment (reynoldmorel) [inline migrations/migrate.cfg:25]: I think this is useful, that way you know when this file was created without going to git

**Review by reynoldmorel — CHANGES_REQUESTED** [review]
Is there a way to point to sql file?

Creating the upgrade funcion and from within the function, specify the file location and then run the migration?

File: migrations/migrate.cfg (hunk 2 above)
Comment (eapg) [reply migrations/migrate.cfg:4]: done

File: migrations/migrate.cfg (hunk 4 above)
Comment (eapg) [reply migrations/migrate.cfg:25]: done

File: migrations/migrate.cfg (hunk 3 above)
Comment (eapg) [reply migrations/migrate.cfg:10]: done

**Pull request closed without merging**